content/doc/book/system-administration/administering-jenkins.adoc

```diff
@@ -0,0 +1,101 @@
+---
+layout: section
+title: Administering Jenkins
+---
+
+== `JENKINS_HOME` directory
+Jenkins needs some disk space to perform builds and keep archives.
+You can check this location from the configuration screen of Jenkins.
+By default, this is set to `~/.jenkins`, but you can change this in one of the following ways:
```

Comment:
Will everyone know what this means -- under $HOME and in a hidden directory that will not show up in plain ls output; you need to use `ls -a`.  *nix people should know this but I'm not so sure Windows people do.